### DISCUSSION TITLE
【Keyball61】プリント基板剥がれについて

### DISCUSSION THREAD

**Discussion by ytakky2014** [Yowkees/keyball#709] [Q&A] [answered]
お世話になっております。

右側にトラックボールを配置するかたちで組んでいます。
LEDは使わない予定です。
テストしていたところ上段右から3番目(18番に相当) だけ反応しないので、確認したところ写真のように基板部が剥がれてしまい、
はんだ不良を起こしているため反応しないようです。
ビルドガイド3-7のように18番をピンセットにて短絡しても反応しなかったためキースイッチ部分のはんだ不良ではないとみています。
![PXL_20241214_143812620](https://github.com/user-attachments/assets/a1e43068-134d-441d-9762-3a34ab78f696)

なにか修正する方法はありますでしょうか？

念のため右側の基板の写真も乗せておきます。
![PXL_20241214_150338048](https://github.com/user-attachments/assets/8b2349fb-93c6-4a16-8315-a8ccc10d4dbb)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

ダイオードの右側のパッドは生きていますか？
ひとまず右側だけ半田付けしていただき、下記写真の緑点どうしをエナメル線等で接続すれば復活できます。
![KB61diodefix](https://github.com/user-attachments/assets/99915f91-b53f-4162-bd58-0e1a964739ad)

もしオレンジ点どうしが導通していないようでしたら、オレンジ点どうしも接続してください。
これで修理できるかと思いますのでお手数ですがご確認お願いいたします。

**Comment by ytakky2014** [comment]
緑点同士をエナメル線で接続したところ動作するようになりました。

ありがとうございました。
